src/App.js

```diff
@@ -147,12 +163,18 @@ function App() {
 
       <section className="blue section">
         <div className="card">
-          <p>
-          The animation abvove shows the effects of a filter on an image in <b>grayscale</b>. A grayscale image has a depth of 1 since it just stores the black/white intensity information of each pixel. 
-          A majority of the images taken today are in color. 
-          These images have multiple matrices stacked on top of each other with each matrix containing pixel values for a specific color. 
-          In an RGB image, for example, there are 3 matrices stacked on top of each other. <b>Thus, we say that an RGB image has a depth of 3.</b> The topmost layer contains the red intensity values of each pixel, followed by the middle matrix that stores information of the green intensities of each pixel and finally, the last layer that stores information about the blue intensities of each pixel.
+          <p>The animation above shows the effects of a filter on an image in <em>grayscale</em> (black and white). </p>
+          <p>A grayscale image has a depth of 1 since it just stores the black/white <em>intensity</em> (brightness) of each pixel. </p>
+          <p> So, as we've already seen, convolution with a greyscale image looks like this: </p>
+          <img src={greyscaleGIF} alt="Greyscale filter animation" id="filterGIF"/>
+          <p> But a majority of the images taken today are in color. </p>
+          <p> These images have multiple matrices stacked on top of each other with each matrix containing pixel values for a specific color.
+          In an <em> RGB </em> image, for example, there are 3 matrices - red, green, and blue - stacked on top of each other.
+          Thus, we say that <em> an RGB image has a depth of 3</em>.
+          The topmost layer contains the red intensity values of each pixel, followed by the middle matrix that stores information of the green intensities of each pixel and finally, the last layer that stores information about the blue intensities of each pixel.
           </p>
+          <p> With three layers instead of one, convolution with an RGB image looks like this: </p>
+          <img src={rgbGIF} alt="RGB filter animation" id="filterGIF" style={{width: '400px'}}/>
```

Comment:
You may want to use columns or some other way to lay out this content - there is a ton of awkward empty space right now. Not merge-blocking if you need to work on other stuff!